qmetaobject/src/lib.rs

```diff
@@ -215,10 +215,32 @@ pub mod webengine;
 /// Module intended for glob import.
 pub mod prelude {
     pub use crate::{
-        qml_register_type, qrc, qt_base_class, qt_method, qt_plugin, qt_property, qt_signal,
-        QAbstractListModel, QByteArray, QColor, QDate, QDateTime, QModelIndex, QObject, QObjectBox,
-        QPointer, QQmlExtensionPlugin, QQuickItem, QQuickView, QRectF, QString, QTime, QVariant,
+        QAbstractListModel,
```

Comment:
I realize this might be controversial, but it was difficult for me to guess what would be the right location for `QEnum` and `qml_register_enum`. If I added the function after `qt_signal` I would have had to re-wrap all the lines. At the expense of an uglier diff in this PR, this list will be more easier to edit in the future (with diffs only showing added/deleted items).

Reply:
I believe actually cargo fmt is wrapping and sorting these.

Reply:
Oh, my bad then. I saw in another thread fmt is not used in this project so I assumed that this was handcrafted. I can reformat it then.

Reply:
Reformatted it. There was quite a bit more formatting that `cargo fmt` applied but I didn't stage those changes.